### PR TITLE
Add extra parameters for Event Api

### PR DESCRIPTION
**Pull request by krichprollsch**
Give the way to pass $top and $skip parameters

### CODE REVIEW

File: src/Api/EventApi.php [hunk 1]
```diff
@@ -131,12 +132,13 @@ public function getCalendarView(
         if (!empty($filter)) {
             $params['$filter'] = $filter;
         }
+            $params['$to'] = 5;
```
Comment (Taluu) [inline src/Api/EventApi.php]: not quite sure this should be here

Comment (krichprollsch) [inline src/Api/EventApi.php]: indeed tks

Comment (krichprollsch) [inline src/Api/EventApi.php]: fixed

**Comment by Taluu** [comment]
thanks @krichprollsch

**Pull request merged**